music_assistant/controllers/config.py

```diff
@@ -66,6 +69,76 @@
 rename = wrap(os.rename)
 
 
+def validate_announcement_chime_url(url: str) -> None:
+    """Validate announcement chime URL format."""
+    if not url or not url.strip():
+        return  # Empty URL is valid (uses default)
+
+    url = url.strip()
+
+    try:
+        parsed = urlparse(url)
+
+        # Check URL scheme
+        if parsed.scheme not in ("http", "https"):
+            raise InvalidDataError("URL must use http:// or https:// protocol")
+
+        # Check if hostname exists
+        if not parsed.netloc:
+            raise InvalidDataError("URL must include a valid hostname")
+
+        # Check for audio file extension
+        path_lower = parsed.path.lower()
+        audio_extensions = (".mp3", ".wav", ".flac", ".ogg", ".m4a", ".aac")
+        if not any(path_lower.endswith(ext) for ext in audio_extensions):
+            raise InvalidDataError(
+                f"URL must point to an audio file ({', '.join(audio_extensions)})"
+            )
+
+    except InvalidDataError:
+        raise  # Re-raise InvalidDataError as-is
+    except Exception as e:
+        raise InvalidDataError(f"Invalid URL format: {e!s}")
+
+
+async def test_announcement_chime_url(url: str) -> None:
+    """Test if the chime URL is accessible."""
+    if not url or not url.strip():
+        return  # Empty URL is valid
+
+    url = url.strip()
+
+    try:
+        timeout = aiohttp.ClientTimeout(total=10)  # 10 second timeout
```

Comment:
never initiate a new aiohttp client session. Always use the global one that lives in the mass class.
That said, I would not do this skip because it will delay the playback. You will know soon enough if it doesnt work